Examples/SwiftPackageManager/OHHTTPStubsDemo/MainViewController.swift

```diff
@@ -0,0 +1,143 @@
+//
+//  ViewController.swift
+//  OHHTTPStubsDemo
+//
+//  Created by Olivier Halligon on 18/04/2015.
+//  Copyright (c) 2015 AliSoftware. All rights reserved.
+//
+
+import UIKit
+import OHHTTPStubs
+import OHHTTPStubsSwift
+
+class MainViewController: UIViewController {
```

Comment:
Couldn't we reuse the same swift source files (and Storyboards and Info.plist) for the SPM example project?
ie just have a different pbxproj which integrates the lib via SPM instead of CocoaPods, but include the same files otherwise? Would avoid duplicating the swift files used by each example app and simplify future maintenance of those.

Except if the imports are different depending on SPM or CP integration? (Though for that we could imagine using `#if` to use one import or the other if they are different)

Reply:
Oh right we still need to use 9.0.0 not 8.1, could have done a "RequestS Changes" for that, don't merge until you at least fix this 😅

Reply:
Probably, but this PR is pretty huge and has been open forever.  I think it's best to handle that later so that we don't have to make people wait for months longer (I simply don't have the time lately for this).

Reply:
Renaming branch PR is here: https://github.com/AliSoftware/OHHTTPStubs/pull/320